lib/auth/webauthn/login_test.go

```diff
@@ -412,22 +410,19 @@ func TestPasswordlessFlow_BeginAndFinish(t *testing.T) {
 
 			// Verify that we recorded user verification requirements in storage.
 			require.Len(t, identity.SessionData, 1)
-			var sd *wanpb.SessionData
+			var sd *wanlib.SessionData
 			for _, v := range identity.SessionData {
 				sd = v // Get SessionData without guessing the key.
 				break
 			}
-			wantSD := &wanpb.SessionData{
+			wantSD := &wanlib.SessionData{
 				Challenge:        sd.Challenge,
 				UserId:           nil,   // aka unset
 				AllowCredentials: nil,   // aka unset
 				ResidentKey:      false, // irrelevant for login
 				UserVerification: string(protocol.VerificationRequired),
 			}
-			if !proto.Equal(sd, wantSD) {
-				diff := cmp.Diff(wantSD, sd)
-				t.Fatalf("SessionData mismatch (-want +got):\n%s", diff)
-			}
+			require.Equal(t, wantSD, sd, "SessionData mismatch")
```

Comment:
Maybe:

```suggestion
			if diff := cmp.Diff(wantSD, sd); diff != "" {
				t.Fatalf("SessionData mismatch (-want +got):\n%s", diff)
			}
```

Closer to the original, plus I like the diff-error better than the requires error... 😊